docs/tutorials/multiscale_deblending.ipynb

```diff
@@ -0,0 +1,584 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Multiscale deblending tutorial\n",
+    "\n",
+    "The idea behind deblending with wavelets is that in theory it should be easier to separate high frequency signals from low frequency signals, since sources are less blended a higher frequencies (the center of most astrophysical sources). In this tutorial two different types of wavelet deblending are attempted, as described below. But first we show the results using the current version of scarlet. This also serves as an example of how to implement your own deblending schemes using scarlet lite."
```

Comment:
I'm not sure the results warrant a notebook, given that we already have starlets in the LSB tutorial

Reply:
This notebook is different because it implements starlets at the blend level, similar to the Stack et al. method but with the additional constraints of monotonicity for each source and the SED/morph factorization while also demonstrating how to use the modular architecture of the lite module to create custom classes. But if you really don't like it here I can remove it and just keep it in a repo on the LSST side.